### PR TITLE
Update distributions.md

### PR DESCRIPTION
**Pull request by CyrusBelsoi**
### What does this PR do? What is the motivation?

Updates to our dist. metrics type offering, new functionality release

### Merge instructions

- [ ] Awaiting a few changes before merge

### CODE REVIEW

File: content/en/metrics/distributions.md [hunk 1]
```diff
@@ -17,12 +17,16 @@ Distributions are a metric type that aggregate values sent from multiple hosts d
...
 
 Distributions provide enhanced query functionality and configuration options that aren't offered with other metric types (count, rate, gauge, histogram):
-* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
+* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points, including stddev) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as:
```
Comment (maycmlee) [inline content/en/metrics/distributions.md:20]:
```suggestion
* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points, including standard deviation) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
```

File: content/en/metrics/distributions.md [hunk 2]
```diff
@@ -17,12 +17,16 @@ Distributions are a metric type that aggregate values sent from multiple hosts d
...
-* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
+* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points, including stddev) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
 
   * **Single percentile value over any timeframe**:
   
      _"What has the 99.9th percentile load time for my application been over the past week?"_
 
+ * **Variance in metrics performance over time**:
+  
+     _"What is the standard deviation (stddev) of my application's CPU consumption over the past 3 months?"_
```
Comment (maycmlee) [inline content/en/metrics/distributions.md:28]:
```suggestion
     _"What is the standard deviation (STDDEV) of my application's CPU consumption over the past 3 months?"_
```

File: content/en/metrics/distributions.md [hunk 3]
```diff
@@ -42,7 +46,7 @@ See the [Developer Tools section][1] for more implementation details.
 
 Like other metric types, such as `gauges` or `histograms`, distributions have the following aggregations available: `count`, `min`, `max`, `sum`, and `avg`. Distributions are initially tagged the same way as other metrics, with custom tags set in code. They are then resolved to host tags based on the host that reported the metric. 
 
-However, you can enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99). Enabling advanced queries also unlocks threshold queries.
+However, you can enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99), including stddev. Enabling advanced queries also unlocks threshold queries.
```
Comment (maycmlee) [inline content/en/metrics/distributions.md:49]:
```suggestion
However, you can enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99), including standard deviation. Enabling advanced queries also unlocks threshold queries.
```

**Review by maycmlee — COMMENTED** [review]
Left a couple of suggestions!

**Review by maycmlee — APPROVED** [review]
@CyrusBelsoi is this good to merge?

File: content/en/metrics/distributions.md [hunk 4]
```diff
@@ -17,12 +17,16 @@ Distributions are a metric type that aggregate values sent from multiple hosts d
...
 
 Distributions provide enhanced query functionality and configuration options that aren't offered with other metric types (count, rate, gauge, histogram):
-* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
+* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points, including standard deviation) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as:
```
Comment (ijkaylin) [inline content/en/metrics/distributions.md:20]: I don't think we should be including standard deviation in this percentiles (). 

```suggestion
* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
```

File: content/en/metrics/distributions.md [hunk 5]
```diff
@@ -17,12 +17,16 @@ Distributions are a metric type that aggregate values sent from multiple hosts d
...
-* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
+* **Calculation of percentile aggregations**: Distributions are stored as DDSketch data structures that represent raw, unaggregated data such that globally accurate percentile aggregations (p50, p75, p90, p95, p99 or any percentile of your choosing with up to two decimal points, including standard deviation) can be calculated across the raw data from all your hosts. Enabling percentile aggregations can unlock advanced query functionalities such as: 
 
   * **Single percentile value over any timeframe**:
   
      _"What has the 99.9th percentile load time for my application been over the past week?"_
 
+  * **Variance in performance over time**:
```
Comment (ijkaylin) [inline content/en/metrics/distributions.md:26]:
```suggestion
  * **Standard deviation over any timeframe
```

File: content/en/metrics/distributions.md [hunk 6]
```diff
@@ -42,7 +46,7 @@ See the [Developer Tools section][1] for more implementation details.
...
 
-However, you can enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99). Enabling advanced queries also unlocks threshold queries.
+However, you can enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99), including standard deviation. Enabling advanced queries also unlocks threshold queries.
```
Comment (ijkaylin) [inline content/en/metrics/distributions.md:49]: Same as above - standard deviation is not a percentile. This phrasing implies that it is 

```suggestion
However, you can enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99). Enabling advanced queries also unlocks threshold queries and the standard deviation.
```

**Review by ijkaylin — CHANGES_REQUESTED** [review]

**Pull request merged**